vault/helm/vault/templates/vault-configmap.yaml

```diff
@@ -53,12 +86,18 @@ data:
     jwt=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
     token=$(vault write auth/kubernetes/login role=admin jwt="$jwt" | grep -m1 '^token' | awk '{ print $2 }')
     vault login -no-print -non-interactive "$token"
+  write-admin-policy.sh: |
+    echo "Writing admin policy"
+    vault policy write admin /policies/admin-policy.hcl
   {{- if .Values.oidc.enabled }}
   enable-oidc.sh: |
     set -euo pipefail
 
     echo "Checking if OIDC is already enabled"
-    if vault auth list | grep -q ^oidc/; then
+    set +e
```

Comment:
@DavidSpek I don't understand why you reverted this test to what it was before my change. My change is much more readable and reduces 4 lines of code into one.

Reply:
Because it didn't work, this changes actually broke the installation of existing users (which also made all subsequent app deployments fail). When I originally created this script I did something similar to what you were doing, but quickly noticed that it failed in a lot of cases. Which is how I ended up with this script.